src/styles/index.scss

```diff
@@ -1,3 +1,142 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  font-size: 12px;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #060b35;
+}
+
+.card {
```

Comment:
card and stars are 2 separate BEM blocks and according to [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) point 11 there should be 1 file for each